routes/api/follows.js

```diff
@@ -0,0 +1,31 @@
+const express = require('express');
+const router = express.Router();
+const createError = require('http-errors');
+const { Follow } = require("../../db/models");
+const {asyncHandler} = require('../utils');
+
+
+
+
+
+router.delete('/follows/:id', asyncHandler(async (req, res) => {
+    if (!res.locals.authenticated) {
+        return next(createError(401)); 
+     } else if (res.locals.user.id === follow.followId && res.locals.authenticated) {
```

Comment:
if `res.locals.user.id !== follow.followerId` we should also return a 401. Also, `follow` does not yet exist. You'll need to query for the follow before checking.